react/src/App.js

```diff
@@ -1,11 +1,17 @@
 import React from 'react'
+import { Router, browserHistory, Route, IndexRoute } from 'react-router';
+import NavBar from './components/NavBar'
+import ParkShow from './containers/ParkShow'
 
 const App = (props) => {
   return(
-     <div>
-       <h1>hello from App</h1>
-     </div>
-  )
+   <Router history={browserHistory}>
+     <Route path='/' component={NavBar} >
+       <IndexRoute component={ParkShow}/>
```

Comment:
The Index Route should be your ParkIndex, not ParkShow. I realize you might be waiting on this from another pair, but for now you can just have this render a "stand-in" blank ParkIndex component for now